products/identity/Incident Response/System Recovery Plan Exercise Template.md

```diff
@@ -0,0 +1,28 @@
+# System Recovery Procedure Exercise Template
+
+## Objective
+
+To ensure that all team members can effectively execute the system recovery procedure.
+
+## {{ service name }} Outage exercise
+
+### Preparation Details
+
+**Responder(s):** {{ name of identity team member(s) }}
+
+**Link to relevant SRP:** {{ github link to SRP document }}
+
+**Link to scenario monitor:** {{ monitor for service }}
+
+**Scenario description:** {{ monitor for service }} is alerting in the [#va-identity-alerts](https://dsva.slack.com/archives/C02SBFQ22RL) channel which indicates {{ service name }} is currently experiencing an outage. The alert has been active for more than one hour.
+
+### Execution
+
+1. Describe the scenario and determine if an incident should be declared.
+    1. Review the definition of an incident document (github link needed) if you are unsure of the qualifications.
+    2. do not declare a real incident at this time, just talk through doing it.
+2. Assign roles and responsibilities to team members for handling the incident.
+3. Describe the process of the service SRP, walk through the steps required
+    1. any identified gaps should be noted for future improvements to the SRP
+4. Conduct a post-exercise review to discuss what went well and what could be improved.
```

Comment:
Do we want to define any sort of retrospective documentation or required follow up docs, or is that not necessary for SRP specifically?

Reply:
I just made [this](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/teams/vsp/teams/Identity/Documentation/Internal_Process/retro_template.md), would this work to link it?

Reply:
Yeah